docs/en/ingest-management/beats-agent-comparison.asciidoc

```diff
@@ -388,7 +388,7 @@ The following table shows a comparison of capabilities supported by {beats} and
 |{y}
 |{n}
 |{y}
-|The Integrations app requires a network connection to the {fleet-guide}/fleet-overview.html#package-registry-intro[Elastic Package Registry]. We are considering an https://github.com/elastic/integrations/issues/1178[on-prem version of EPR]. {fleet}-managed {agent}s require a connection to our artifact repository for agent binary upgrades. These are not required for standalone {agent}s or {beats}.
+|The Integrations app requires a network connection to the {fleet-guide}/fleet-overview.html#package-registry-intro[{package-registry}], or a {integrations-developer-guide}/air-gapped.html#air-gapped-diy-epr[self-managed deployment of the {package-registry}]. {fleet}-managed {agent}s require a connection to our artifact repository for agent binary upgrades. These are not required for standalone {agent}s or {beats}.
```

Comment:
I couldn't find a way to link to the air-gapped page. I tried these formats:
* `<<air-gapped>>` (I guess this doesn't work because this is in a different guide)
* `{integrations}/air-gapped.html[]`
* `{integrations-developer-guide}/air-gapped.html[]`

How could I define a link to this page?

I also wonder if this page should be moved to the `ingest-management` guide, as is not so related to integrations development.

Reply:
Ah I see the problem. We don't have an attribute for this book in our shared attributes file yet. I've opened a PR here to add it: https://github.com/elastic/docs/pull/2298

However, I don't think the air gapped docs should live in the integrations developer guide. Users need to know this info, too. I would move it to the Fleet and Elastic Agent Guide, probably under https://www.elastic.co/guide/en/fleet/7.16/integrations.html.

Reply:
So once that PR is merged, you would use: {integrations-devguide}/filename.html[Topic name]

Reply:
I have tried to move the page, and now there are broken links, should I leave some kind of placeholder (or redirect)? in the old page?

Reply:
Thanks for the help btw! :slightly_smiling_face: